src/algorithm/ATOMIC-Cuda.cpp

```diff
@@ -10,20 +10,24 @@
 
 #include "RAJA/RAJA.hpp"
 
+#include <iostream>
+
 #if defined(RAJA_ENABLE_CUDA)
 
 #include "common/CudaDataUtils.hpp"
 
 #include <cub/block/block_reduce.cuh>
 #include <cub/warp/warp_reduce.cuh>
 
-#include <iostream>
+#endif  // RAJA_ENABLE_CUDA
```

Comment:
I don't understand the addition #if-#endif macro sections. Why doesn't an #if at the top of the file and the corresponding #endif at the bottom of the file work for your changes?

Reply:
I think I found a way around this.